doc/devguide/README.md

```diff
@@ -0,0 +1,14 @@
+# Suricata Developer Guide
+
+This directory contains the Suricata Developer Guide. The
+[Sphinx Document Generate](http://sphinx-doc.org) is used to build the
```

Comment:
`Generate` -> `Generator`. I realize this is a typo brought over from the user guide. Should probably fix it there as well (as a separate commit).